### ISSUE TITLE
Reduce plugin not

### ISSUE THREAD

**Issue by opn** [fedwiki/wiki-plugin-reduce#7]
I'm having the following issue with the reduce problem which I do not recall from my earlier install - <img width="707" alt="screen shot 2015-07-26 at 13 44 18" src="https://cloud.githubusercontent.com/assets/80854/8893957/9e72ef7a-339c-11e5-8bc2-0f20219ce971.png">

In general I'm finding it difficult to follow the use of the reduce and rollup plugins, and think it woud be useful to make a step-by-step video tutorial - taking a simple financial budgeting example familiar to everyone.

If we have method plugins on several pages that calculate individual elements how do we bring them together on another page to make calculations?

**Comment by paul90** [comment]
Looks like the error is
```
"plugin error" TypeError: slider.slider is not a function
Stack trace:
emit@http://rodwell.me/js/jquery-1.11.3.min.js line 2 > eval:271:1
[24]</g.doPlugin/<@http://rodwell.me/client.js:4:3587
[24]</g.getPlugin/<@http://rodwell.me/client.js:4:2273
[24]</g.getScript/<@http://rodwell.me/client.js:4:2095
m.Callbacks/j@http://rodwell.me/js/jquery-1.11.3.min.js:2:27304
m.Callbacks/k.fireWith@http://rodwell.me/js/jquery-1.11.3.min.js:2:28122
x@http://rodwell.me/js/jquery-1.11.3.min.js:5:22109
.send/b@http://rodwell.me/js/jquery-1.11.3.min.js:5:26030
 client.js:4:3636

TypeError: slider.slider is not a function
```

**Comment by WardCunningham** [comment]
That is a jQuery UI widget. Perhaps the Reduce plugin expected jQuery UI to be loaded by the static html header? I notice that a fairly recent install works: [asia.wiki.org](http://asia.wiki.org/about-reduce-plugin.html)

It loads a custom version:
```
<script src='/js/jquery-ui-1.10.1.custom.min.js' type='text/javascript'></script>
```
unlike the most recent npm install:
```
<script src='/js/jquery-ui/1.11.4/jquery-ui.min.js' type='text/javascript'></script>
```
I think that is the only place we use jQuery UI so it should be browserified with the plugin, no?

**Comment by WardCunningham** [comment]
I am reminded just how cool the Reduce plugin is. It reflects on its own contents, finds the names of pages containing calculations, and then reevaluates those calculations from three different pages it loaded itself using less than half of the 17ms animation frame. Just to show off, double clicking the slider thumb rewrites the calculation as spreadsheet formulas.
![image](https://cloud.githubusercontent.com/assets/12127/8894478/ca7e323a-336c-11e5-9d41-d18e96258561.png)

I wrote this to support my work with Michael Mehaffy. My conclusion was that it could be made much easier, much faster and much more powerful. See [Reduce Revisited](http://ward.asia.wiki.org/reduce-revisited.html)

**Comment by paul90** [comment]
> I think that is the only place we use jQuery UI so it should be browserified with the plugin, no?

Sure it's used in a few places

I'll update the jQuery UI library in the client in the morning...

**Comment by paul90** [comment]
fixed by + wiki-client@0.5.1
